src/api/apiClient.js

```diff
@@ -62,15 +62,15 @@ export class ApiClient {
     //  genres
     getAllGenres = async () =>
         await this.postGlobalResource(
-            `/genre/movie/list` + this.createFilterString(),
+            '/genre/movie/list' + this.createFilterString(),
```

Comment:
Please use template strings instead of concatenation

Reply:
Fixed

Reply:
Fix commit: https://github.com/JIeCoRyI3/the-movie-db/pull/17/commits/cccd090d6f5479ab7182aaba488c84ea94eab5d2